File: senz-client-samples/python/utils/image_utils.py
import base64

# Convert image to string
def imageToString(fileName):
	with open(fileName, "rb") as imageFile:
		return base64.b64encode(imageFile.read())
		
# Convert string to image
def stringToImage(str, fileName):
	fh = open(fileName, "wb")
	fh.write(base64.b64decode(str.encode()))
	fh.close()

File: senz-client-samples/ruby/utils/imageUtils.rb
require "base64"
#filename = "sample.jpg"

def imageToString(filename)
  encodedImage = Base64.encode64(File.open(filename,"rb").read)
  return encodedImage
end

def stringToImage(str , filename)
  #saving image
  File.open(filename,"wb") do |file|
    file.write(Base64.decode64(str))
  end
end
